.github/workflows/ci.yml

```diff
@@ -271,19 +271,22 @@ jobs:
         echo ::endgroup::
         echo ::group::Crashing Test Logs
         # See if we don't have a crash that went away
-        python -m pytest -n 1 -m "crashes" -sv --max-worker-restart 512 | tee test_crashed.log 2>&1
+        python -m pytest -n 1 -m "xfail" --runxfail -sv --max-worker-restart 512 | tee test_crashed.log 2>&1
         echo ::endgroup::
         echo ::group::XFAIL Test Logs
+        # Comment out all xfails but the ones that have a run=False condition.
+        find . -name "*.py" -exec sed -i '/run=False/!s/^ *@mark.xfail\(.*\)/#&/' {} \;
         # See if we don't have an xfail that went away
-        python -m pytest -m "not crashes" --runxfail -s  | tee test_xfailed.log 2>&1
+        python -m pytest -m "xfail" --runxfail -sv  | tee test_xfailed.log 2>&1
```

Comment:
You have to select `not xfail`

Reply:
or comment out the `run=False` ones only

Reply:
We run the `not xfail` in a few lines below. Here we remove all the xfail flags but the ones that have `run=False` and we run them. But I think I understand what you mean - good point. Let me open a new PR.